ddl/partition.go

```diff
@@ -1521,7 +1539,8 @@ func (w *worker) onExchangeTablePartition(d *ddlCtx, t *meta.Meta, job *model.Jo
 		return ver, errors.Wrapf(err, "failed to notify PD the label rules")
 	}
 
-	ver, err = updateSchemaVersion(d, t, job)
+	nt.ExchangePartitionInfo.ExchangePartitionFlag = false
```

Comment:
I think it's better to use `nt.ExchangePartitionInfo = nil` instead.

Reply:
done.